vnext/ReactUWP/Views/ReactControl.cpp

```diff
@@ -339,6 +340,10 @@ void ReactControl::PrepareXamlRootView(XamlView const &rootView) {
     children.Clear();
 
     auto newRootView = winrt::Grid();
+    auto perspectiveTransform3D = winrt::Windows::UI::Xaml::Media::Media3D::PerspectiveTransform3D();
+    perspectiveTransform3D.Depth(850);
```

Comment:
>850 [](start = 33, length = 3)

Is this the same value that the other platforms use?

Reply:
@StephenLPeters this is the value that the RNTester app uses for its transform sample. Note that RN lets you set a perspective on a per-object basis whereas here we're setting it for the whole scene. In speaking with Chris this is ok since we just want to make sure we don't have an default orthographic projection and we mimic RN as much as we can. Setting the perspective on an individual view or animating the perspective is outside the scope of this particular issue/PR :)

Reply:
@Stephen L Peters this is the value that the RNTester app uses for its transform sample. Note that RN lets you set a perspective on a per-object basis whereas here we're setting it for the whole scene. In speaking with Chris this is ok since we just want to make sure we don't have an default orthographic projection and we mimic RN as much as we can. Setting the perspective on an individual view or animating the perspective is outside the scope of this particular issue/PR :)

---
In reply to: [340369650](https://github.com/microsoft/react-native-windows/pull/3551#discussion_r340369650) [](ancestors = 340369650)

Reply:
Do we have an issue tracking the full work of supporting perspective on a per-object basis to align with RN?  -- Setting a value to make RNTester work seems like a bad thing to do. -- as it hides the fact that it doesn't work.  -- Unless we think this is a better default than what XAML normally has?

Reply:
XAML has a default orthographic projection so objects that are far don't look smaller than objects that are closer and all lines perpendicular to the Z plane are drawn as points (they're all parallel, no vanishing point).
This change creates a default perspective with one vanishing point which is what RN expects/has. 
Created #3560 to track adding per-object perspective. There's also #2680 which tracks allowing it to be animated (this will be tricky since there isn't a façade for perspective depth in xaml)

---
In reply to: [340425407](https://github.com/microsoft/react-native-windows/pull/3551#discussion_r340425407) [](ancestors = 340425407)